### ISSUE TITLE
Font chunks in webpack watch mode

### ISSUE THREAD

**Issue by eranshmil** [SirPole/google-fonts-plugin#8]
Thanks for this great plugin!

I'm running my webpack with watch flag, and each time I change something the chunk list is getting bigger. It doesn't seem to make duplications but it makes the log hard to read.

![image](https://user-images.githubusercontent.com/9721664/53684471-73082800-3d16-11e9-9322-91ba03f0cf6e.png)

Here is my configuration file:
```js
'use strict';

const path = require('path');
const CleanWebpackPlugin = require('clean-webpack-plugin');
const CopyWebpackPlugin = require('copy-webpack-plugin');
const MiniCssExtractPlugin = require('mini-css-extract-plugin');
const GoogleFontsPlugin = require('google-fonts-plugin');

const srcFolder = 'src';
const distFolder = 'dist';
const srcFileExtensions = ['.ts', '.js'];
const copyIgnoredFiles = ['*.ts', '*.js', '*.scss', '*.html'];

module.exports = (env, argv) => ({
  devtool: 'sourcemap',
  entry: {
    content: [`./${srcFolder}/content.ts`, `./${srcFolder}/content.scss`]
  },
  output: {
    path: path.join(__dirname, distFolder),
    filename: '[name].js'
  },
  module: {
    rules: [
      {
        test: /\.(js|ts)$/,
        use: 'ts-loader',
        exclude: /node_modules/
      },
      {
        test: /\.scss$/,
        use: [MiniCssExtractPlugin.loader, 'css-loader', 'sass-loader']
      },
      {
        test: /\.png$/i,
        use: 'url-loader'
      },
      {
        test: /\.html$/i,
        use: 'html-loader?exportAsEs6Default'
      }
    ]
  },
  plugins: [
    new CleanWebpackPlugin(distFolder),
    new CopyWebpackPlugin([
      {
        from: '**/*',
        context: srcFolder,
        ignore: copyIgnoredFiles
      }
    ]),
    new MiniCssExtractPlugin({
      filename: '[name].css',
      chunkFilename: '[id].css'
    }),
    new GoogleFontsPlugin({
      fonts: [
        {
          family: 'Heebo',
          variants: ['400', '700'],
          subsets: ['hebrew']
        }
      ],
      formats: ['woff', 'woff2']
    })
  ],
  resolve: {
    extensions: srcFileExtensions
  }
});
```

Versions:
```
OS: macOS 10.13.6
node: 11.3.0
yarn: 1.13.0
webpack: 4.29.5
webpack-cli: 3.2.3
google-fonts-plugin: 4.0.2
```

**Comment by SirPole** [comment]
Hey! I've managed to fix this finally, released in [4.1.0](https://www.npmjs.com/package/google-fonts-plugin) 🚀

**Comment by eranshmil** [comment]
Thanks!
